src/main/java/com/pusher/client/connection/websocket/WebSocketConnection.java

```diff
@@ -303,6 +307,12 @@ public void run() {
         }, reconnectInterval, TimeUnit.SECONDS);
     }
 
+    // Received error codes 4000 >= 4099 indicate we shouldn't attempt reconnection
```

Comment:
I'm not clear what `4000 >= 4099` precisely means. What about the notation the docs use `4000-4099`?